data/templates/new_game_template

```diff
@@ -0,0 +1,18 @@
+[global]
+interactive_player="1"
+win_condition="scripting/win_conditions/endless_game.lua"
```

Comment:
Do we need the path? I guess win conditions are always at the same path?

Reply:
They can also be in add-ons, so you might want to set e.g. 
`win_condition="addons/100-wells.wad/init.lua"`

Should I change this to be consistent with `init`, i.e. specifying either a filename in `data/scripting/win_conditions` or an add-on name?

Reply:
That would be better imho.